### PR TITLE
PR template preview and changelog update

### PR DESCRIPTION
**Pull request by rhrmo**
This PR implements a new PR template and updates CHANGELOG

## Categories

- [ ] Bugfix
- [ ] Enhancement
- [ ] Backporting
- [X] Others (CI, Infrastructure, Documentation)

## Privacy

There are no sensitive data in the newly collected information.

## Breaking Changes

No

## References

https://issues.redhat.com/browse/CCXDEV-9402

### CODE REVIEW

File: cmd/changelog/main.go [hunk 1]
```diff
@@ -17,11 +17,13 @@ import (
...
 // norevive
 const (
-	BUGFIX      = "Bugfix"
-	ENHANCEMENT = "Enhancement"
-	OTHER       = "Others"
-	BACKPORTING = "Backporting"
-	MISC        = "Misc"
+	BUGFIX          = "Bugfix"
+	DATAENHANCEMENT = "Data Enhancement"
+	ENHANCEMENT     = "Enhancement"
```
Comment (tremes) [inline cmd/changelog/main.go:22]: I think we can remove this.

Comment (rhrmo) [reply cmd/changelog/main.go:22]: It is needed for the regex so that Enhancement category PRs can be collected and then generated in Data Enhancement.

Comment (tremes) [reply cmd/changelog/main.go:22]: Ahh OK I see...

File: cmd/changelog/main.go [hunk 2]
```diff
@@ -17,11 +17,13 @@ import (
...
+	FEATURE         = "Feature"
+	OTHER           = "Others"
+	BACKPORTING     = "Backporting"
```
Comment (tremes) [inline cmd/changelog/main.go:25]: I am just wondering if we need this item at all.....because it doesn't seem to be used in the CHANGELOG, right?

**Comment by tremes** [comment]
Thank you!
/lgtm

**Comment by tremes** [comment]
This is minor change of the PR template and changelog script. 
/label px-approved
/label qe-approved
/label docs-approved

**Pull request merged**